src/activities/babymatch/DragListItem.qml

```diff
@@ -103,6 +103,16 @@ Item {
             property Item currentTargetSpot
             property bool pressedOnce
             property bool parentIsTile : parent == tile ? true : false
+            
```

Comment:
Here the line is empty but contains spaces. It is a good practice to not put space before an end of line.